packages/hash/api/src/index.ts

```diff
@@ -132,13 +137,20 @@ const main = async () => {
     port: graphApiPort,
   });
 
+  await ensureSystemOrgAccountIdExists({ graphApi, logger });
+
   await ensureSystemTypesExist({ graphApi, logger });
 
+  await ensureSystemOrgExists({
+    graphApi,
+    logger,
+  });
+
   await ensureSystemEntitiesExists({ graphApi, logger });
```

Comment:
Do we want to just group these into a single method at this point? (We can still have 4 exported methods if we think they'd be helpful in tests for example to have fine-grained control)

Reply:
That makes sense, I've already introduced such a methods in the tests so have moved and made use of it in the backend in https://github.com/hashintel/hash/pull/1596/commits/2f497441d396eeb58da8e3e6740225ad1fd5fbf6